table-api/src/main/java/io/deephaven/api/agg/Aggregation.java

```diff
@@ -291,6 +293,27 @@ static FirstRowKey AggFirstRowKey(String resultColumn) {
         return FirstRowKey.of(resultColumn);
     }
 
+    /**
+     * <p>
+     * Create a {@link io.deephaven.api.agg.spec.AggSpecFormula formula} aggregation with the supplied {@code formula}.
+     * This variant requires the formula to provide the output column name and specific input column names in the
+     * following format:
+     * </p>
+     * {@code
+     * AggFormula("output_col=(input_col1 + input_col2) * input_col3")
+     * }
+     *
+     * @param formula The {@link AggSpecFormula#formula() formula} to use to produce the output column
+     * @return The aggregation
+     */
+    static Formula AggFormula(String formula) {
+        // Parse the supplied formula for the output column name and formula
+        final Selectable column = Selectable.parse(formula);
```

Comment:
I think it would be appropriate to introduce `Formula.parse` for this logic. I would also argue that it should _not_ (ab)use `Selectable.parse` even though the logic is rather similar; (if we really want to reduce code duplication, we can create a package private impl that can essentially parse "<lhs>=<rhs>" and can invoke `BiConsumer<String, String>`...)